docker-compose.yml

```diff
@@ -67,11 +68,17 @@ services:
       - ./public:/usr/src/code/public
       - ./src:/usr/src/code/src
       - ./debug:/tmp
+      - ./dev:/usr/local/dev
     depends_on:
       - mariadb
       - redis
       # - clamav
       - influxdb
+    entrypoint: 
```

Comment:
Why do we need to change the entrypoint?

Reply:
"-e" flag is needed for debugger to work, which shouldn't be in production